VACMS-17068 News Releases web components

randimays opened the pull request:
## Summary
Add web components where possible on the News Releases search results page and the detail pages.

I also added a new section to the README for this repository and updated the `DRUPAL_ADDRESS` that no longer works.

## Related issue(s)
- https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17068

## Testing done
Tested locally on these pages:
<details><summary>/minneapolis-health-care/news-releases/</summary>


### Article titles
<img width="1400" alt="Screenshot 2024-04-30 at 3 58 18 PM" src="https://github.com/department-of-veterans-affairs/content-build/assets/19175324/7413247d-143e-4b51-a026-33fa9e691cfe">
<img width="350" alt="Screenshot 2024-04-30 at 4 24 22 PM" src="https://github.com/department-of-veterans-affairs/content-build/assets/19175324/7d37dd5e-94a9-4edf-8424-2afc2fd2e00f">

</details>

<details><summary>/minneapolis-health-care/news-releases/st-james-va-community-clinic-ribbon-cutting-ceremony/</summary>

### Email address for media contact
<img width="1294" alt="Screenshot 2024-04-30 at 4 30 50 PM" src="https://github.com/department-of-veterans-affairs/content-build/assets/19175324/37643d76-6f30-4cb0-a0d9-3d6d7d178aee">

### Active link
<img width="1296" alt="Screenshot 2024-04-30 at 4 31 40 PM" src="https://github.com/department-of-veterans-affairs/content-build/assets/19175324/b592df0f-2656-41e3-86ec-f7da7849fd8f">

</details>

<details><summary>
/boston-health-care/news-releases/va-new-england-announces-new-deputy-executive-director/</summary>

### Download Press Release button

<img width="1607" alt="Screenshot 2024-04-30 at 4 32 57 PM" src="https://github.com/department-of-veterans-affairs/content-build/assets/19175324/00681b49-f63b-4533-aab2-65ab63f28cf2">

### Media download links

<img width="1610" alt="Screenshot 2024-04-30 at 4 35 05 PM" src="https://github.com/department-of-veterans-affairs/content-build/assets/19175324/078f6c37-689c-4a71-8518-79f3d3065f4d">


</details>

<details><summary>
/boston-health-care/news-releases/6-million-intimate-partner-violence-intervention-study-approved/</summary>
<img width="1608" alt="Screenshot 2024-04-30 at 4 37 39 PM" src="https://github.com/department-of-veterans-affairs/content-build/assets/19175324/003c0a1a-0dfc-4c29-a5bc-6dd6e73a3223">

</details>

## Acceptance criteria
- [x] Link component has been added
- [ ] Requires accessibility review

jilladams commented on README.md:
````diff
@@ -45,7 +47,7 @@ Authentication credentials are stored in a local .gitignore'd file `.env`. An ex
 ```
 # .env file contents
-DRUPAL_ADDRESS=https://content-build-medc0xjkxm4jmpzxl3tfbcs7qcddsivh.ci.cms.va.gov
+DRUPAL_ADDRESS=https://cms-8ry6zt2asg946vdfuiryyamuc9gkuyzc.demo.cms.va.gov/
````
@department-of-veterans-affairs/cms-infrastructure FYI that this URL change has been noted by Sitewide engineers as something that changed invisibly and has impacted local dev in content-build. Want to make sure you're aware, in case of future updates, that this README will need the same update if the URL changes in future.

jilladams replied:
Related discussion: https://dsva.slack.com/archives/C52CL1PKQ/p1714670201799399?thread_ts=1714669962.085509&cid=C52CL1PKQ

laflannery commented:
Looks good to me!

eselkin reviewed (approved):
LGTM. Was the README stuff from main?

randimays commented:
@eselkin No, I added the README updates. Our DRUPAL_ADDRESS was out of date from some long-ago Tugboat weirdness, so I updated it. I've had a few teams approach me lately needing to run content-build, so I gave the whole thing a once-over.
